packages/aws-cdk/lib/api/deployments/cfn-api.ts

```diff
@@ -240,7 +240,7 @@ async function uploadBodyParameterAndCreateChangeSet(
       role: executionRoleArn,
     });
   } catch (e: any) {
-    await ioHelper.notify(IO.DEFAULT_TOOLKIT_DEBUG.msg(e));
+    await ioHelper.notify(IO.DEFAULT_TOOLKIT_DEBUG.msg(String(e)));
```

Comment:
seems like a small bug i came across: the message must be a string as we eventually call `.trim()` on it